components/prombench/manifests/benchmark/5_nginx-ingress-routes.yaml

```diff
@@ -12,9 +12,9 @@ spec:
       paths:
       - backend:
           serviceName: prometheus-test-{{ normalise .RELEASE }}
-          servicePort: 80
+          servicePort: prom-web
```

Comment:
where does this name come from?
if it is another manifest file I would say best to keep these separate to avoid braking when updating one manifest without knowing it is referred by in another file.

Reply:
`prom-web` is defined in https://github.com/prometheus/prombench/blob/master/components/prombench/manifests/benchmark/3_prometheus-test.yaml. I will revert the changes in this file.